### DISCUSSION TITLE
How to run MOOSE in parallel mode?

### DISCUSSION THREAD

**Discussion by abc-hy** [idaholab/moose#26085] [Q&A Modules: Phase field]
Hi everyone,

Does anyone know how can I run MOOSE simulation in parallel mode? Do I need to change anything in the input file?

Thank you very much.
Best,

**Comment by grmnptr** [comment]
No need to chane anything, maybe your preconditioner depending on your problem.

```
mpirun -n 100 myexecutable input.i
```
